fdbserver/storageserver.actor.cpp

```diff
@@ -2859,6 +2911,8 @@ ACTOR Future<Void> fetchKeys(StorageServer* data, AddingShard* shard) {
 		                   keys,
 		                   true); // keys will be available when getLatestVersion()==transferredVersion is durable
 
+		// Note it does not leave this thread until this point, since it receives a pointer to FetchInjectionInfo.
```

Comment:
what is "it" referring to?

Reply:
Rephrased to `Note that since it receives a pointer to FetchInjectionInfo, the thread does not leave this actor until this point.` and also resolving @Daniel-B-Smith 's comment about "thread".